test/timesheet.js

```diff
@@ -0,0 +1,223 @@
+const db = require('../source/database');
+const Timesheet = require('../source/timesheet');
+const collectionName = 'Timesheets';
+
+describe('Timesheet.js Tests:', function(){
+    
+    // Connects to database before testing
+    before(async function() {
+        await db.myConnect();
+    });
+
+    // Closes database connection after testing
+    after(async function(){
+        await db.myClose();
+    });
+
+    describe('Timesheet Validation Tests:', function(){
+
+        let timesheetValues = {
+            _id : db.createMongoID(),
+            date : new Date(),
+            employeeID : db.createMongoID(),
+            jobNumber : 12345,
+            jobID : db.createMongoID(),
+            estCrewSize : 3,
+            estCrewHours : 8,
+            tmCrewSize : 0,
+            tmCrewHours : 0,
+            lunchTaken : true,
+            jobFinished : true,
+            offHauled : false,
+            yardsHauled : 0,
+            notes : 'Test Notes'
+        };
+
+        it('Timesheet Validation - Happy Path', async function(){
```

Comment:
All of these 'it' s do not need to be asynchronous once you change validate() to not be asynchronous.

Reply:
Fixed.